src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -278,3 +279,35 @@ private function getClassNameFromIncomplete(\__PHP_Incomplete_Class $value)
         return $array['__PHP_Incomplete_Class_Name'];
     }
 }
+
+namespace Symfony\Component\HttpKernel\Exception;
```

Comment:
there is an issue when putting this in the same file than the other exception: the ``Symfony\Component\HttpKernel\Exception\FlattenException`` cannot be autoloaded anymore, so this is a BC break for people using it directly